### ISSUE TITLE
[P2][Regression][Sourcing][Accessibility][Multi-input] : 10627 not working anymore

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#12434]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
If the developer is passing title attribute value then instead of screenreader reading "value-help, menu-popup, button", the screenreader should read the string thats passed to title.
This was already fixed in issue https://github.com/SAP/fundamental-ngx/issues/10627  , but looks like its no more working.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular: 15
Core/ platform: 0.43.43

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
1) pls use [link](https://stackblitz.com/edit/angular-9qxxca-9mvc5b?file=src%2Fapp%2Fplatform-table-preserved-state-example.component.html,src%2Fapp%2Fplatform-table-preserved-state-example.component.ts,package.json)
2) Open voiceover
3) move the focus to the value help button
4) the screenreader should read "Test 124" as per value of title property

#### Please provide relevant source code (if applicable).
NA

#### Please provide stackblitz example(s).
https://stackblitz.com/edit/angular-9qxxca-9mvc5b?file=src%2Fapp%2Fplatform-table-preserved-state-example.component.html,src%2Fapp%2Fplatform-table-preserved-state-example.component.ts,package.json

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
This was fixed sometime as part of https://github.com/SAP/fundamental-ngx/issues/10627

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by mikerodonnell89** [comment]
It looks like my past solution was to tie the aria-label to the title, but really, sending a title attribute to an element should not have an effect on the screenreader. Also we've been advised against relying on title attributes because they are not screenreader friendly, nor keyboard only user friendly. So I am going to make some changes to the multi-input to decouple the title from the aria behaviors

**Comment by I543348** [comment]
> It looks like my past solution was to tie the aria-label to the title, but really, sending a title attribute to an element should not have an effect on the screenreader. Also we've been advised against relying on title attributes because they are not screenreader friendly, nor keyboard only user friendly. So I am going to make some changes to the multi-input to decouple the title from the aria behaviors, and give the developer total control over what is read by the screenreader when the input or the addon button are focused

Thanks @mikerodonnell89 , is that going to be a new property to make the aria-label read by screenreader as you are now going to decouple title ? Pls confirm,
Thanks

**Comment by mikerodonnell89** [comment]
> > It looks like my past solution was to tie the aria-label to the title, but really, sending a title attribute to an element should not have an effect on the screenreader. Also we've been advised against relying on title attributes because they are not screenreader friendly, nor keyboard only user friendly. So I am going to make some changes to the multi-input to decouple the title from the aria behaviors, and give the developer total control over what is read by the screenreader when the input or the addon button are focused
> 
> Thanks @mikerodonnell89 , is that going to be a new property to make the aria-label read by screenreader as you are now going to decouple title ? Pls confirm, Thanks

Correct, a new input for the multi-input is added (`addOnButtonAriaLabel`) that will be read when the add-on button is focused

**Comment by I543348** [comment]
> > > It looks like my past solution was to tie the aria-label to the title, but really, sending a title attribute to an element should not have an effect on the screenreader. Also we've been advised against relying on title attributes because they are not screenreader friendly, nor keyboard only user friendly. So I am going to make some changes to the multi-input to decouple the title from the aria behaviors, and give the developer total control over what is read by the screenreader when the input or the addon button are focused
> > 
> > 
> > Thanks @mikerodonnell89 , is that going to be a new property to make the aria-label read by screenreader as you are now going to decouple title ? Pls confirm, Thanks
> 
> Correct, a new input for the multi-input is added (`addOnButtonAriaLabel`) that will be read when the add-on button is focused

Thanks @mikerodonnell89 , this sounds good to me

**Comment by I543348** [comment]
Hi @mikerodonnell89 , i see this is fixed only for https://github.com/SAP/fundamental-ngx/releases/tag/v0.52.2-rc.33
could you pls downport the issue to ng15 as well?

**Comment by droshev** [comment]
@I543348 the fox for ng15 - https://github.com/SAP/fundamental-ngx/pull/12438
